seqr/views/react_app.py

```diff
@@ -22,7 +22,7 @@ def no_login_main_app(request, *args, **kwargs):
     user_token = kwargs.get('user_token')
     if user_token:
         initial_json['newUser'] = _get_json_for_user(User.objects.get(password=user_token))
-    elif not request.user.is_anonymous():
+    elif not request.user.is_anonymous:
```

Comment:
The `is_anonymous` attribute has changed from a function to a variable in the new version of the WSGI request.